php/streaming_stdin.php

```diff
@@ -0,0 +1,12 @@
+<?php
+// Script reads streaming input text and then prints capitalized string to stdout
+
+while (true) {
```

Comment:
Let's make this loop look like the one in `php/stdin.php`

Reply:
Oh duh. I should have done that earlier. Doing it now!